src/testRunner/unittests/config/convertCompilerOptionsFromJson.ts

```diff
@@ -420,6 +420,70 @@ namespace ts {
             );
         });
 
+        it("Convert empty string option of moduleSuffixes to compiler-options ", () => {

```

Comment:
Need some tests when module specifier already has extension
Another test when extension is ".json"

Reply:
Added more tests to cover these scenarios.